PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -334,6 +334,7 @@ BEGIN
     CONTROL         "Find dialog remains open after search that outputs to results window",IDC_CHECK_FINDDLG_ALWAYS_VISIBLE, "Button", BS_AUTOCHECKBOX | WS_TABSTOP,37,40,350,10
     CONTROL         "Confirm Replace All in All Opened Documents",IDC_CHECK_CONFIRMREPLOPENDOCS, "Button", BS_AUTOCHECKBOX | WS_TABSTOP,37,55,350,10
     CONTROL         "Replace: Don't move to the following occurrence", IDC_CHECK_REPLACEANDSTOP, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 37, 70, 350, 10
+    CONTROL         "Find in files search result: show only one entry per found line", IDC_CHECK_SHOWONCEPERFOUNDLINE, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 37, 85, 350, 10

```

Comment:
But it doesn't just affect "Find in files", it affects any search that outputs to "Search results" window, e.g. "Find all in current doc".  Suggest change wording to "Find All search result: ...".

Reply:
Or maybe even just "Search Result window: show only one entry per found line", to strongly emphasize it's for that particular window.

Reply:
@pryrt 
Done: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/11808/commits/a051f6c465ad050d7f9e255f8b13eaff46181791